### PR TITLE
Emit the crash issue template for more exceptions and crashes

### LINKED ISSUE #5668: Investigate #5495 (crash without a provided template)
See https://github.com/PyCQA/pylint/issues/5495#issuecomment-1011022169

### PR DESCRIPTION
**Pull request by DanielNoord**
- [x] Add yourself to CONTRIBUTORS if you are a new contributor.
- [x] Add a ChangeLog entry describing what your PR does.
- [x] If it's a new feature, or an important bug fix, add a What's New entry in
      `doc/whatsnew/<current release.rst>`.
- [x] Write a good description on what the PR does.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Description

Closes #5668.

Was unsure about the exception to raise, but I think this makes sense. Also added typing 😄

### CODE REVIEW

**Comment by DanielNoord** [comment]
At some point I'm going to remove these Windows test runners 😡

**Comment by Pierre-Sassoulas** [comment]
> At some point I'm going to remove these Windows test runners

They are "costing" 10 times what a linux job cost too,. But would advise against removing them 😄 Let's just ignore this one, it's obviously a server issue.

File: pylint/lint/pylinter.py [hunk 1]
```diff
@@ -1212,14 +1212,17 @@ def _astroid_module_checker(self):
...
+    def get_ast(
+        self, filepath: str, modname: str, data: Optional[str] = None
+    ) -> nodes.Module:
         """Return an ast(roid) representation of a module or a string.
 
         :param str filepath: path to checked file.
         :param str modname: The name of the module to be checked.
         :param str data: optional contents of the checked file.
         :returns: the AST
         :rtype: astroid.nodes.Module
```
Comment (Pierre-Sassoulas) [inline pylint/lint/pylinter.py:1224]:
```suggestion
```
isn't that generated from the typing ?

File: tests/lint/test_utils.py [hunk 2]
```diff
@@ -31,3 +35,16 @@ def test_get_fatal_error_message() -> None:
     assert python_path in msg
     assert crash_path in msg
     assert "open an issue" in msg
+
+
+def test_issue_template_on_fatal_errors(capsys: pytest.CaptureFixture) -> None:
```
Comment (Pierre-Sassoulas) [inline tests/lint/test_utils.py:40]: Nice test 👍

File: pylint/lint/pylinter.py (hunk 1 above)
Comment (DanielNoord) [reply pylint/lint/pylinter.py:1224]: Partially, but the additional text isn't. No reason to remove it, somebody took the effort to write it sometime.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**